org.msweet.htmldoc.appdata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.msweet.htmldoc</id>
+  <name>HTMLDOC</name>
+  <summary>HTML Conversion Software</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  <description>
+    <p>HTMLDOC is a program that reads HTML and Markdown source files or web pages and generates corresponding EPUB, HTML, PostScript, or PDF files with an optional table of contents.</p>
+  </description>
+  <launchable type="desktop-id">org.msweet.htmdoc.desktop</launchable>
+</component>
```

Comment:
file is missing `<content_rating>`, `<releases>`, `<screenshots>`

can we assume to will be part of the next upstream release too?

Reply:
I don't post screenshots, are they necessary?

The `content_rating` and `releases` elements were not part of the generated XML from the appdata web form... I guess I can convert the current (markdown) changelog into a releases/release tree, but I'm not sure what to put for the rating since this is just a conversion utility so any rating depends on the HTML files/web pages that are being converted.

Reply:
https://github.com/flathub/flathub/wiki/App-Requirements#Appstream shows that you can get the tool that Flathub uses (I see the instructions don't actually tell you to *install* it, but you can get that from flathub). That'll help, and yes screenshots are required for applications that declare themselves to be a desktop-application.

You can answer the OARS questions https://hughsie.github.io/oars/generate.html from the perspective of your app and anything that you provide in it - not what users bring to the table in their own files. So in your case I'd imagine the answer would be 'there's nothing scary here.

https://github.com/flathub/flathub/wiki/AppData-Guidelines has some more tips and best practice.